packages/main/src/Tokenizer.ts

```diff
@@ -228,11 +232,16 @@ class Tokenizer extends UI5Element {
 		});
 	}
 
-	onAfterRendering() {
-		this._scrollEnablement.scrollContainer = (this.expanded || !this.narrowContentDom) ? this.expandedContentDom : this.narrowContentDom;
+	async onAfterRendering() {
```

Comment:
Again, this seems beyond TS refactoring, should such a big change be part of this PR?